src/components/Footer/Footer.css

```diff
@@ -31,7 +31,7 @@
     font-size: 16px;
     font-style: normal;
     font-weight: 400;
-    line-height: 19px;
+    line-height: 19.36px;
```

Comment:
i think sometimes i rounded in my pr's also like this (19 for 19.36) because at one point someone on practicum staff told us that was fine but will start trying to be exact just in case!

Reply:
I rounded it too, but found that it would actually shifted the layout by a couple of pixels. Personally I think that level of exactness is unneccessary, but they want it to be "pixel perfect"